civis/civis.py

```diff
@@ -13,6 +13,34 @@
 
 
 def find(object_list, filter_func=None, **kwargs):
+    """Return the elements from ``object_list`` that satisfy the filters.
```

Comment:
An example would be nice.  It doesn't seem very clear what  `object_list : iterable` means as is.

Reply:
The function as-is seems pretty generic, and can take an arbitrary iterable of any objects. I'll try to come up with something and put it in the `Examples` section of the docstring.